cypress/e2e/course/createEditAndDeleteCourse.feature

```diff
@@ -4,6 +4,15 @@ Feature: Course - To add and delete a course by the teacher
 
   As a teacher I want to create a new course and want to delete the newly created test course/room so that list of courses/rooms can be cleaned and not full with the newly created test courses/rooms.
 
+  @stable_test
+  Scenario: as a pre-condition teacher deletes undeleted tests
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to rooms overview
+    Then I delete all courses named 'Cypress Test Creation and Deletion'
+    And I do not see the course 'Cypress Test Creation and Deletion' on the room overview page
```

Comment:
we are trying avoid using 'And' in the feature file.
so this line can be as follows:
'Then I do not see the course 'Cypress Test Creation and Deletion' on the room overview page'